code/game/objects/structures/aliens.dm

```diff
@@ -270,7 +266,8 @@
 /obj/structure/alien/weeds/Destroy()
 	STOP_PROCESSING(SSobj, src)
 	QUEUE_SMOOTH_NEIGHBORS(src)
-	playsound(loc, pick('sound/effects/alien_resin_break2.ogg','sound/effects/alien_resin_break1.ogg'), 50, FALSE)
+	if(!silent_removal)
+		playsound(loc, pick('sound/effects/alien_resin_break2.ogg','sound/effects/alien_resin_break1.ogg'), 50, FALSE)
```

Comment:
in a later PR, we need to be moving this out of `Destroy()` as its not really what the proc was intended for